### PR TITLE
Issue-55 Response method name is different than request method name

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/Method.java [hunk 1]
```diff
@@ -31,30 +31,85 @@
...
+        @Override
         public String getName() {
             return name;
         }
         
         @Override
         public boolean equals(Object obj) {
-            if (obj instanceof Name) {
-                Name other = (Name) obj;
+            if (obj instanceof RequestName) {
```
Comment (mbaechler) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/Method.java]: ResponseName

Comment (aduprat) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/Method.java]: OK

**Comment by mbaechler** [comment]
:+1:

**Comment by mbaechler** [comment]
merged into JAMES-1644

**Pull request closed without merging**